code/modules/mapping/random_engines.dm

```diff
@@ -0,0 +1,44 @@
+/datum/map_template/random_engine
```

Comment:
again- copypasting shit without knowing how it works

Reply:
This is obviously to save the engine templates, do you think i would just throw them into the random rooms file ?